seqr/fixtures/1kg_project.json

```diff
@@ -82,6 +82,20 @@
         "created_by": null,
         "last_modified_date": "2017-03-12T23:52:55.846Z",
         "name": "test category name",
+        "projects": [
+            1, 2
+        ]
+    }
+},
+    {
+    "model": "seqr.projectcategory",
+    "pk": 4,
+    "fields": {
+        "guid": "PC000003_test_category_demo",
```

Comment:
I do not think you meant to commit this here

Reply:
Hmm..., it is for other test. I will remove it.

Reply:
I found that this row of the data is trying to satisfy the search with "allProjectFamilies" option. Line  101 in variant_search_api.py is as below:
```
            omit_projects = ProjectCategory.objects.get(name='Demo').projects.all()
```
It uses the parameter "name='Demo' " to search the database. Therefore, I created a new row in the ProjectCategory table of 1kg_project with a name "Demo".
I'm not sure what the purpose of the line and why it uses a literal value "Demo".

Reply:
Ah got it